src/Template/Layout/default.ctp

```diff
@@ -1,11 +1,5 @@
 <!DOCTYPE html>
-
-<?php
-echo $this->fetch(
-    'html',
-    sprintf('<html lang="%s" class="no-js">', read('App.language'))
-);
-?>
+<html lang="<?= \Locale::getPrimaryLanguage(\Cake\I18n\I18n::locale()) ?>">
```

Comment:
Setting Content Language Header and this is very imporant fir the browser nowadays, css hyphens uses those. Great stuff.